glossary.md

```diff
@@ -4,18 +4,21 @@ title: Glossary
 nav_order: 8
 published: true
 ---
+
 # Glossary of Common Terms
+
 {: .no_toc}
 This page contains a selection of key terms important to understanding the structure of data within Data Commons.
 
 ## Term List
-{: .no_toc}
 
-* TOC
-{:toc}
+{: .no_toc}
 
+- TOC
+  {:toc}
 
 ### [Cohort](https://datacommons.org/browser/CohortSet)
+
```

Comment:
Alse remove the extra space here between heading and {: #id}, that way the id will be tagged correctly

Reply:
Done